HelpSource/Classes/ScopeView.schelp

```diff
@@ -185,8 +196,7 @@ s.waitForBoot(
     syn1=SynthDef("test1", { arg bus, bufnum;
                     var z;
                     z = In.ar(bus,2);
-                    // ScopeOut writes the audio to the buffer
-				//                     ScopeOut.ar(z, bufnum);
+        // ScopeOut2 writes the audio to the buffer
```

Comment:
The indentation here doesn't match the rest.

Reply:
Ok, no problem – I tried to incorporate the changes in the commit below. Is it all right?